### ISSUE TITLE
Video checkers rewards issue

### ISSUE THREAD

**Issue by benblack769** [PettingZoo-Team/Multi-Agent-ALE#9]
Looking at results for standard average reward calculation for pettingzoo, I got:

```
    "all_mean_abs_rewards": 0.4964171358286417,
    "all_mean_rewards": -0.4964171358286417,
    "all_prop_non_zero": 0.4964171358286417,
    "avg_episode_len": 200002.0,
    "mean_abs_rewards": {
      "first": 0.23681213187868122,
      "second": 0.2596050039499605
    },
```

Indicating a problem.

**Comment by benblack769** [comment]
The problem being that reward should be received maybe 1 in several hundred frame, and is actually occurring every other frame

**Comment by benblack769** [comment]
This issue has been solved and is released in MA-ALE 0.1.10
